azerothcore/azerothcore-wotlk
# fix(DB/Creature): Fix '(Master) Fire Eater' / '(Master) Flame Eater' spawns and SAI based on sniffs

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:

Thanks @heyitsbench for providing the sniffs!

## Tests Performed:

This PR has been:


## How to Test the Changes:



.event start 1

Fire Eater
.go c 25951

Flame Eater
.go c 25972

Master Fire Eater
.go c 202769

Master Flame Eater
.go c 202751

observe
- equips bottle or torches at the beginning
- juggling torches or fire beathing (chosen randomly) every 100-120s
- celebrants cheering at the end of each little show
- thanks the celebrants with bowing - emote
- unequips items again

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
